test/integration/local/test_dummy.py

```diff
@@ -47,6 +47,6 @@ def test_install_requirements(capsys):
 
     stdout = capsys.readouterr().out
 
-    assert "Installing collected packages: pyfiglet, train.py" in stdout
-    assert "Successfully installed pyfiglet-0.8.post1 train.py-1.0.0" in stdout
+    assert "Installing collected packages: pyfiglet" in stdout
+    assert "Successfully installed pyfiglet-0.8.post1" in stdout
```

Comment:
regex it to remain version agnostic?

Reply:
The `requirements.txt` file used in this test has the version pinned, so this should be okay.

Reply:
makes sense. it is possible maintainer may remove release (and may be more probable, given it's a *post release).